project/Dependencies.scala

```diff
@@ -19,72 +19,88 @@ object Dependencies {
-    val akkaSlf4j        = "com.typesafe.akka"     %% "akka-slf4j"                        % V.akka
-    val pureconfig       = "com.github.pureconfig" %% "pureconfig"                        % V.pureconfig
-    val akkaHttpMetrics  = "fr.davit"              %% "akka-http-metrics-datadog"         % V.akkaHttpMetrics
+    val collectorPayload = "com.snowplowanalytics"  % "collector-payload-1"                % V.collectorPayload
+    val badRows          = "com.snowplowanalytics"  %% "snowplow-badrows"                  % V.badRows
+    val trackerCore      = "com.snowplowanalytics"  %% "snowplow-scala-tracker-core"       % V.tracker
+    val trackerEmitterId = "com.snowplowanalytics"  %% "snowplow-scala-tracker-emitter-id" % V.tracker
+    val scopt            = "com.github.scopt"       %% "scopt"                             % V.scopt
+    val akkaHttp         = "com.typesafe.akka"      %% "akka-http"                         % V.akkaHttp
+    val akkaStream       = "com.typesafe.akka"      %% "akka-stream"                       % V.akka
+    val akkaSlf4j        = "com.typesafe.akka"      %% "akka-slf4j"                        % V.akka
+    val pureconfig       = "com.github.pureconfig"  %% "pureconfig"                        % V.pureconfig
+    val akkaHttpMetrics  = "fr.davit"               %% "akka-http-metrics-datadog"         % V.akkaHttpMetrics
+    val scalaJava8Compat = "org.scala-lang.modules" %% "scala-java8-compat"                % V.scalaJava8Compat
 
     // Scala (test only)
-    val specs2            = "org.specs2"        %% "specs2-core"         % V.specs2   % Test
-    val akkaTestkit       = "com.typesafe.akka" %% "akka-testkit"        % V.akka     % Test
-    val akkaHttpTestkit   = "com.typesafe.akka" %% "akka-http-testkit"   % V.akkaHttp % Test
-    val akkaStreamTestkit = "com.typesafe.akka" %% "akka-stream-testkit" % V.akka     % Test
+    val specs2            = "org.specs2"            %% "specs2-core"                   % V.specs2         % Test
+    val specs2Cats        = "org.specs2"            %% "specs2-cats"                   % V.specs2Cats     % Test
+    val specs2CE          = "com.codecommit"        %% "cats-effect-testing-specs2"    % V.specs2CE       % Test
+    val catsEffect        = "org.typelevel"         %% "cats-effect"                   % V.catsEffect     % Test
+    val akkaTestkit       = "com.typesafe.akka"     %% "akka-testkit"                  % V.akka           % Test
+    val akkaHttpTestkit   = "com.typesafe.akka"     %% "akka-http-testkit"             % V.akkaHttp       % Test
+    val akkaStreamTestkit = "com.typesafe.akka"     %% "akka-stream-testkit"           % V.akka           % Test
+    val testcontainers    = "com.dimafeng"          %% "testcontainers-scala-core"     % V.testcontainers % Test
+    val eventGen          = "com.snowplowanalytics" %% "snowplow-event-generator-core" % V.eventGen       % Test
```

Comment:
I like the current setup, where integration tests are not part of a specific module, but a module on their own. I'm specifically thinking of all tests that are about some core, sink-independent functionality, which will otherwise have to be included in a specific project, like `stdout`. It seems to me that it would make it more difficult to find a specific test that way.

Having said that, the current organisation of tests is also not ideal. Eg, in `StdOut` spec we have tests that are not specifically about the `stdout` sink. I want to come up with a better organisation, but I was waiting to have more tests, so that we can more easily see what makes sense.